lib/resty/ldap/filter.lua

```diff
@@ -121,17 +117,11 @@ local filter = P{
         ((_'ATTRIBUTE_VALUE' * _'WILDCARD') +
         (_'WILDCARD' * _'ATTRIBUTE_VALUE' * _'WILDCARD') +
         (_'WILDCARD' * _'ATTRIBUTE_VALUE')) / function(...)
-            local args = pack(...)
-            local s = ''
-            for i = 1, args.n, 1 do
-                local value = args[i]
-                if type(value) == "table" then
-                    s = s .. value.attribute_value
-                else
-                    s = s .. value
-                end
+            local s = {}
+            for _, value in ipairs({...}) do
```

Comment:
We can use select to avoid extra table, like https://github.com/apache/apisix/blob/63f8a67f2c6e02cd196cdb46b4af2d83085703cc/apisix/core/response.lua#L65?

Reply:
fixed